packages/components/src/public/QueryColumn.ts

```diff
@@ -246,6 +247,7 @@ export class QueryColumn implements IQueryColumn {
     declare userEditable: boolean;
     declare validValues: string[];
     // declare versionField: boolean;
+    declare valueExpression: string;
```

Comment:
If this is now furnished by the server, then we should consider updating the interface `QueryColumn` in `@labkey/api`.

Reply:
done